src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -1,52 +1,37 @@
 package cromwell.engine.backend.local
 
-import java.io.{Writer, BufferedWriter, File, FileWriter}
-
 import cromwell.binding.values.WdlValue
 import cromwell.binding.{Call, TaskOutput}
-import cromwell.engine.backend.Backend
 import cromwell.engine.SymbolStore
+import cromwell.engine.backend.Backend
+import cromwell.util.FileUtil
+import cromwell.util.FileUtil._
 
 import scala.language.postfixOps
+import scala.sys.process._
 import scala.util.Try
 
 class LocalBackend extends Backend {
 
   override def executeCommand(commandLine: String, call: Call, taskOutputs: Set[TaskOutput], symbolStore: SymbolStore): Map[String, Try[WdlValue]] = {
 
-    implicit class FlushingAndClosingWriter(writer: Writer) {
-      /** Convenience method to flush and close in one shot. */
-      def flushAndClose() = {
-        writer.flush()
-        writer.close()
-      }
-    }
-
-    def buildFileAndWriter(filename: String): (File, Writer) = {
-      val file = new File(filename)
-      val writer = new BufferedWriter(new FileWriter(file))
-      (file, writer)
-    }
-
-    import sys.process._
-
-    // TODO make these proper temp files, but for now it's preferable
-    // TODO to leave them in a deterministic place.
-    val (stdoutFile, stdoutWriter) = buildFileAndWriter("stdout.txt")
-    val (stderrFile, stderrWriter) = buildFileAndWriter("stderr.txt")
-    val (commandFile, commandWriter) = buildFileAndWriter("command.txt")
+    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout")
+    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr")
+    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command")
 
     commandWriter.write(commandLine)
     commandWriter.flushAndClose()
 
     s"/bin/bash ${commandFile.getAbsolutePath}" ! ProcessLogger(stdoutWriter write, stderrWriter write)
 
-    Seq(stdoutWriter, stderrWriter).foreach { _.flushAndClose() }
+    Seq(stdoutWriter, stderrWriter).foreach {
```

Comment:
I was going to say to put this on one line but apparently this was intentional :)